app/assets/stylesheets/notifications.scss

```diff
@@ -0,0 +1,18 @@
+@import "application";
+
+#notification-inbox-container {
+  background-color: #d3d3d3;
```

Comment:
Shouldn't this be put in a global CSS variable?

Reply:
Feel free to do so.

Reply:
I added a `$lightGrey` variable in the `Application.scss`.